src/awsexplorer/activation.ts

```diff
@@ -34,47 +36,53 @@ import { copyNameCommand } from './commands/copyName'
 import { loadMoreChildrenCommand } from './commands/loadMoreChildren'
 import { checkExplorerForDefaultRegion } from './defaultRegion'
 import { RegionNode } from './regionNode'
+import { extensionSettingsPrefix } from '../shared/constants'
+import { CredentialsStore } from '../credentials/credentialsStore'
+
+let didTryAutoConnect = false
 
 /**
- * Activate AWS Explorer related functionality for the extension.
+ * Activates the AWS Explorer UI and related functionality.
  */
-
-export async function activate(activateArguments: {
+export async function activate(args: {
     awsContext: AwsContext
     context: vscode.ExtensionContext
     awsContextTrees: AwsContextTreeCollection
     regionProvider: RegionProvider
     toolkitOutputChannel: vscode.OutputChannel
     remoteInvokeOutputChannel: vscode.OutputChannel
 }): Promise<void> {
-    const awsExplorer = new AwsExplorer(
-        activateArguments.context,
-        activateArguments.awsContext,
-        activateArguments.regionProvider
-    )
+    const awsExplorer = new AwsExplorer(args.context, args.awsContext, args.regionProvider)
 
     const view = vscode.window.createTreeView(awsExplorer.viewProviderId, {
         treeDataProvider: awsExplorer,
         showCollapseAll: true,
     })
-    activateArguments.context.subscriptions.push(view)
+    ext.context.subscriptions.push(view)
```

Comment:
Nit: does moving away from `args` to `ext` mean that we can dump `args.context` from the constructor?